Add new auth config option for SignalR (#22932)
dotnet/AspNetCore.Docs: aspnetcore/signalr/authn-and-authz.md

::: moniker-end

::: moniker range="< aspnetcore-6.0"

> [!NOTE]

> > If a token expires during the lifetime of a connection, the connection will continue to work. LongPolling and ServerSentEvents connections will fail on subsequent requests if they don't send newer access tokens.

Reviewer: ```suggestion
> If a token expires during the lifetime of a connection, the connection continues to work. `LongPolling` and `ServerSentEvent` connections fail on subsequent requests if they don't send new access tokens.
```

Reviewer: Avoid FT Future Tense

Reviewer: `LongPolling` and `ServerSentEvent`  are probably wrong but they're not English words that will MT (machine translate)

::: moniker-end

::: moniker range=">= aspnetcore-6.0"

> [!NOTE]

> > If a token expires during the lifetime of a connection, by default the connection will continue to work. LongPolling and ServerSentEvents connections will fail on subsequent requests if they don't send newer access tokens. If you want connections to be closed when the authentication token expires, that can be enabled with [CloseOnAuthenticationExpiration](xref:signalr/configuration#advanced-http-configuration-options).

Reviewer: ```suggestion
> If a token expires during the lifetime of a connection, by default the connection continues to work. `LongPolling` and `ServerSentEvent` connections fail on subsequent requests if they don't send new access tokens. For connections to close when the authentication token expires, set [CloseOnAuthenticationExpiration](xref:signalr/configuration#advanced-http-configuration-options).
```
